### DISCUSSION TITLE
About comments

### DISCUSSION THREAD

**Discussion by ghost** [community/community#28375] [Other Feature Feedback, Questions, & Ideas] [answered]
Is there an option to auto-delete GitHub comments after a period of time?

Is there also an option to delete automatically old comments edits (revisions)?

**Comment by airtower-luna** [comment, marked as answer]
> Is there an option to auto-delete GitHub comments after a period of time?

There isn't.

> Is there also an option to delete automatically old comments edits (revisions)?

This sounds like you might mean *commits* rather than comments. The answer is also no, Git is meant to keep a persistent history. You can still edit the history if you really want to, but it *will* break things if you have shared the repository with others. See [Git Tools - Rewriting History](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History) if you want to learn how to do it.
